src/framework/theme/components/tabset/tabset.component.ts

```diff
@@ -279,7 +290,14 @@ export class NbTabComponent {
           class="tab">
         <a href (click)="$event.preventDefault()" tabindex="-1" class="tab-link">
           <nb-icon *ngIf="tab.tabIcon" [config]="tab.tabIcon"></nb-icon>
-          <span *ngIf="tab.tabTitle" class="tab-text">{{ tab.tabTitle }}</span>
+          <!-- If there is a label template, use it. -->
+          <ng-template [ngIf]="tab.templateLabel">
+            <ng-template [cdkPortalOutlet]="tab.templateLabel"></ng-template>
```

Comment:
Use `*ngTemplateOutlet` instead of portals. And then revert `src/framework/theme/components/cdk/*` changes.